website/docs/cloud-docs/workspaces/settings/health-assessments.mdx

```diff
@@ -0,0 +1,64 @@
+---
+page_title: Health Assessments - Workspaces - Terraform Cloud
+description: >-
+  Enable automatic jobs to check for lifecycle concerns, like whether your real-world infrastructure matches
+  your Terraform state file.
+tfc_only: true
+---
+
+# Health Assessments
+
+-> **Note:** Health Asessments (formerly Drift detection) is in beta and is available in the paid **Business** tier. Refer to [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) for details and  [provide feedback](https://forms.gle/BwjgvrKQcLbgqK3f7) about how we can improve this feature.
+
+The term health assessment is used to describe the regular, automatic inspection of a workspace to surface resource lifecycle concerns. Presently, this includes infrastructure drift detection but may include other concerns in the future.
+
+Infrastructure drift means that your real-world infrastructure no longer matches the configuration in your Terraform state file. Drift occurs when a user modifies resources outside of the Terraform workflow. For example, a colleague may update resource configuration directly in the cloud provider console during a production incident, changing the resource attributes from those in tracked in your state file.
+
+> **Hands On:** Try our [Manage Resource Drift](https://learn.hashicorp.com/tutorials/terraform/resource-drift) tutorial on HashiCorp Learn to review how to detect and resolve drift.
+
+Health assessments is a workspace-level setting. When enabled, Terraform Cloud will automatically run a health assessment for that workspace every 24 hours. The assessment checks the actual settings of your infrastructure against those tracked in your workspace's state file.
```

Comment:
we want to stay away from future-facing statements like "we will add more in the future." I know it's a bit odd because drift detection is the only one, so I massaged it a bit --> thoughts?

```suggestion
Health assessment describes the regular, automatic inspection of a workspace to surface resource lifecycle concerns. Drift detection is a type of health assessment.

Infrastructure drift means that your real-world infrastructure no longer matches the configuration in your Terraform state file. Drift occurs when a user modifies resources outside of the Terraform workflow. For example, a colleague may update resource configuration directly in the cloud provider console during a production incident, changing the resource attributes from those in tracked in your state file.

> **Hands On:** Try our [Manage Resource Drift](https://learn.hashicorp.com/tutorials/terraform/resource-drift) tutorial on HashiCorp Learn to review how to detect and resolve drift.

When enabled, Terraform Cloud automatically runs a health assessment for your workspace every 24 hours. The assessment checks the actual settings of your infrastructure against those tracked in your workspace's state file.
```